src/pages/product-listing/productListing.css

```diff
@@ -0,0 +1,71 @@
+.sidebar {
+    padding-left: 2rem;
+    width: 15vw;
+    gap: 2rem;
+    color: var(--black-text);
+  }
+  
+  .container-one,
+  .range-amount {
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  .container-two input {
+    width: 100%;
+  }
+  .container-two {
+    gap: 1rem;
+  }
+  .container-three,
+  .container-four,
+  .input-container {
+    gap: 0.5rem;
+  }
```

Comment:
try to be consistent on whether to leave space or not after writing certain styling as after line 12 you have given a line space but same is not the case at line-16 & line-19